docs/workforce-management-framework/topics/con-datasync-cloud-module.adoc

```diff
@@ -0,0 +1,42 @@
+= {WFM-RC-NameLong} Datasync Module
+
+As with all synchronisation tasks Datasync Module has cloud and client sides. This is a implementation of 
+the Feedhenry Sync Framework for advanced usage see the 
+link:{WFM-RC-SyncFrameworkURL}{WFM-RC-Branch}/docs[sync framework documentation]. 
+The primary funtion of Datasync is to synchronise mobile data between mobile and a backend server.
+The key features of Datasync are as follows:
+
+- Allows mobile apps to use and update data offline (local cache)
+- Provides a mechanism to manage bi-directional data synchronization from multiple Client Apps via the Cloud App and into back-end data stores
+- Allows data updates (that is, deltas) to be distributed from the Cloud App to connected clients
+- Provides facilities for managing data collisions from multiple updates in the cloud
```

Comment:
"data collisions" - what exactly is this? Suggestion: "Provides facilities for managing <brief explanation of data collisions> when this occurs across multiple updates in the cloud.

or

Leave what is there and add "For more information on data collisions, please see <appropriate link>"

Reply:
same as above